src/sass/_utils.scss

```diff
@@ -31,3 +31,25 @@
 }
 
 // sass-lint:enable no-important
+
+@keyframes sgAnimationFadeIn {
+  0% {
+    opacity: 0;
+  }
+
+  100% {
+    opacity: 1;
+  }
+}
+
+.sg-animation-fade-in-slow {
+  animation-name: sgAnimationFadeIn;
+  animation-duration: 600ms;
+  animation-timing-function: ease-in;
+}
+
+.sg-animation-fade-in-fast {
+  animation-name: sgAnimationFadeIn;
+  animation-duration: 400ms;
+  animation-timing-function: ease-in;
```

Comment:
I found only 3 usages of this particular easing in application which is kind of wrong. Animation principles sugges easing out or in out like for things we're showing, can we please discuss? :)

Reply:
ah probably you meant usage of these: 
![image](https://user-images.githubusercontent.com/8572321/107676689-b789f580-6c99-11eb-9788-67be34f8834b.png)